wallet/client.go

```diff
@@ -12,6 +12,17 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+type AddressType string
```

Comment:
Introduced AddressType to be used in other places so we don't use the strings directly.